src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj

```diff
@@ -31,10 +31,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Amqp" Version="[2.2.0, 3.0.0)" />
-    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.1.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="[1.0.1, 2.0.0)" />
```

Comment:
Going from 1.1.0-preview to 1.0.1.
This is a downgrade, meaning features from 1.1.0-preview are not uesd?

Reply:
`1.1.0-preview` was first and `1.0.1` was introduced _after_ (https://github.com/Azure/azure-sdk-for-net/issues/4121).
Asking as I'm using this for MSI and there are no release notes or PR/issue to determine what has changed in 1.1.0-prevew. Perhaps you folks know.